### DISCUSSION TITLE
Idea : setting page's buttons should close on click

### DISCUSSION THREAD

**Discussion by ace-cltr** [Dun-sin/Whisper#607] [Ideas]
buttons on settings pop-up are not closing or updating any changes

![settings-buttons](https://github.com/Dun-sin/Whisper/assets/81556737/c7d8a576-c98c-4c7e-a226-28f32e393ec6)
![Screenshot 2024-02-15 162806](https://github.com/Dun-sin/Whisper/assets/81556737/7a724a85-71bb-4fb0-8c49-ebeb9685d1f8)

Additional information
none

🥦 Browser
Google Chrome

👀 Have you checked if this issue has been raised before?
 I checked and didn't find similar issue

🏢 Have you read the Contributing Guidelines?
 I have read and understood the rules in the [Contributing Guidelines](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md)

Are you willing to work on this issue ?
No

**Comment by Dun-sin** [comment]
Why should it close on click?
